lib/python/ship_callsign.py

```diff
@@ -1,42 +1,36 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-Reads the ship_callsign text file in the same directory and returns a
+Reads the ship_callsign from platform vocab and returns
```

Comment:
You're returning all codes & names from the platform vocab, not just ships, so why not call these functions `platform_name_list` and `platform_name`, and just put them in the `platform_name_vocab` module?

Otherwise, if you do want to have just the ship codes &  names, you need to filter the vocab list.